cmd/client/txgen/main.go

```diff
@@ -89,166 +95,88 @@ func main() {
 	if peerPubKey == nil {
 		panic(fmt.Errorf("generate key error"))
 	}
-
+	shardID := *shardIDFlag
+	var shardIDs []uint32
```

Comment:
no need for shard IDs array is no multi-shard support.

Reply:
This is not resolved.

Reply:
It is for this `txGen.Client = client.NewClient(txGen.GetHost(), shardIDs)`

Reply:
Then if you want to change the txgen to be single-shard. You should change that interface too.

Reply:
@rlan35 The txgen works for all shards. See image I posted. I have not investigated why.

Reply:
I can change the interface however.

Reply:
Let me do that cleanly in next PR. Added the issue: https://github.com/harmony-one/harmony/issues/846

Reply:
ok